### PR TITLE
updated glossary for v1

### PR DESCRIPTION
**Pull request by FelipeAdachi**
## Description

updated glossary for v1

### General Checklist

* [ ] Tests added for this feature/bug
      if it was a bug, test must cover it.
* [ ] Conform by the style guides, by using formatter
* [ ] Documentation updated
* [ ] (optional) Please add a label to your PR

### CODE REVIEW

File: python/docs/glossary.md [hunk 1]
```diff
@@ -16,38 +16,41 @@ In batch mode, whylogs processes a dataset in batches.
...
-#### whylogs file
-[NEED UPDATING] The **whylogs output** is available in the following formats: protobuf, JSON, and flat. Protobuf is a lightweight binary
-format that maps one-to-one with the memory representation of a whylogs object. JSON displays the protobuf data in JSON
-format. Flat outputs multiple files with both CSV and JSON content to represent different views of the data, including
-histograms, upper-bound, lower-bound, and frequent values. To apply advanced transformation on whylogs, we recommend
-using Protobuf.
-
 #### Metric
-TBD New world
+The summary statistics of a dataset is composed by a collection of metrics. Metrics are defined according to its namespaces. Some of the whylogs' default namespaces are: `counts`, `types`, `distribution`, `cardinality`, `frequent items/frequent strings`. By default, a column of a given data type will have a defined set of metrics assigned to it. The metrics to be tracked are customizable according to data type or column name (see [Schema Configuration](examples/basic/Schema_Configuration.ipynb)) . New metrics can also be created in a custom fashion, according to the need of the user (see TBD).
```
Comment (murilommen) [inline python/docs/glossary.md]: there's an extra space before the period here, after the Schema Configuration hyperlink

File: python/docs/glossary.md [hunk 2]
```diff
@@ -16,38 +16,41 @@ In batch mode, whylogs processes a dataset in batches.
...
+
 
 ##### Metric component
-NEED TO DEFINE HERE
+A metric component is the smallest unit for a metric.A metric might consist of multiple components. An example is ints metric, which consists of min and max metric components. The calculation of components could be independent or could be coupled with other components.
+
+
+#### whylogs file
+
+A whylogs-generated profile can be stored as a protobuf binary file. Protobuf is a lightweight binary
```
Comment (murilommen) [inline python/docs/glossary.md]: I would also state that Protobuf = protocol buffers and that it is a serialization technology, with link to its docs, for users who aren't familiar with it

**Review by murilommen — CHANGES_REQUESTED** [review]
Very good work, Felipe. one small thing i'd extend there

File: python/docs/glossary.md (hunk 1 above)
Comment (FelipeAdachi) [reply python/docs/glossary.md]: resolved in latest update

File: python/docs/glossary.md (hunk 2 above)
Comment (FelipeAdachi) [reply python/docs/glossary.md]: resolved in latest update. thanks!

**Review by andyndang — APPROVED** [review]
LGTM

**Review by murilommen — APPROVED** [review]

**Pull request merged**